os/net/mac/tsch/tsch-schedule.c

```diff
@@ -260,8 +301,10 @@ tsch_schedule_remove_link(struct tsch_slotframe *slotframe, struct tsch_link *l)
       if(l == current_link) {
         current_link = NULL;
       }
-      LOG_INFO("remove_link %u %u %u %u ",
-             slotframe->handle, l->link_options, l->timeslot, l->channel_offset);
+      LOG_INFO("remove_link sf=%u opt=%s ts=%u ch=%u addr=",
+               slotframe->handle,
+               print_link_options(l->link_options),
```

Comment:
shall we add link type here as well?

Reply:
Updated. I have no idea why it was not there in the original version...

Reply:
most prob was just a mistake :)